proto/execution_stats.proto

```diff
@@ -11,49 +11,6 @@ import "proto/stat_filter.proto";
 
 package execution_stats;
 
```

Comment:
moved this to `remote_execution.proto` to break a circular dep (this file depends on remote_execution.proto)